<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyGUI</class>
 <widget class="QWidget" name="MyGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyGUI</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(255, 255, 255)
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Copperplate Gothic Light</family>
         <pointsize>22</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 94, 109);
font: 22pt &quot;Copperplate Gothic Light&quot;;</string>
       </property>
       <property name="text">
        <string>Movie watchlist</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="userButton">
       <property name="font">
        <font>
         <family>Copperplate Gothic Light</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>OpenHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0.494318, y1:0.227, x2:0.449, y2:1, stop:0 rgba(104, 227, 222, 255), stop:1 rgba(255, 255, 255, 255));
font: 16pt &quot;Copperplate Gothic Light&quot;;
color: rgb(0, 94, 109);
height:70px;</string>
       </property>
       <property name="text">
        <string>User</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="watchlistButton">
       <property name="font">
        <font>
         <family>Copperplate Gothic Light</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>OpenHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0.494318, y1:0.227, x2:0.449, y2:1, stop:0 rgba(104, 227, 222, 255), stop:1 rgba(255, 255, 255, 255));
font: 16pt &quot;Copperplate Gothic Light&quot;;
color: rgb(0, 94, 109);
height:70px;</string>
       </property>
       <property name="text">
        <string>Watchlist</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="adminButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Copperplate Gothic Light</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>OpenHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0.494318, y1:0.227, x2:0.449, y2:1, stop:0 rgba(104, 227, 222, 255), stop:1 rgba(255, 255, 255, 255));
font: 16pt &quot;Copperplate Gothic Light&quot;;
color: rgb(0, 94, 109);
height:70px;</string>
       </property>
       <property name="text">
        <string>Admin</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
